tests/modules/programs/nix-your-shell/enable-shells.nix

```diff
@@ -0,0 +1,52 @@
+{ pkgs, config, lib, ... }:
+
+{
+  programs = {
+    nix-your-shell = {
+      enable = true;
+      enableFishIntegration = true;
+      enableNushellIntegration = true;
+      enableZshIntegration = true;
+    };
+    fish.enable = true;
+    nushell.enable = true;
+    zsh.enable = true;
+  };
+
+  # Needed to avoid error with dummy fish package.
```

Comment:
where is `fish` being stubbed?

Reply:
Seems fish ran into other issues when being stubbed (complain about `fish_indent`), and I didn't see any other tests stubbing it, so I removed that.